consensus/objs/ovs.go

```diff
@@ -132,24 +131,24 @@ func (b *OwnValidatingState) MarshalCapn(seg *capnp.Segment) (mdefs.OwnValidatin
 	return bh, nil
 }
 
-func (b *OwnValidatingState) PTOExpired() bool {
+func (b *OwnValidatingState) PTOExpired(proposalStepTO time.Duration) bool {
 	rs := b.RoundStarted
-	return rs+int64(constants.ProposalStepTO)/constants.OneBillion < time.Now().Unix()
+	return rs+int64(proposalStepTO)/constants.OneBillion < time.Now().Unix()
 }
 
-func (b *OwnValidatingState) PVTOExpired() bool {
+func (b *OwnValidatingState) PVTOExpired(preVoteStepTO time.Duration) bool {
 	rs := b.PreVoteStepStarted
-	return rs+int64(constants.PreVoteStepTO)/constants.OneBillion < time.Now().Unix()
+	return rs+int64(preVoteStepTO)/constants.OneBillion < time.Now().Unix()
 }
 
-func (b *OwnValidatingState) PCTOExpired() bool {
+func (b *OwnValidatingState) PCTOExpired(preCommitStepTO time.Duration) bool {
 	rs := b.PreCommitStepStarted
-	return rs+int64(constants.PreCommitStepTO)/constants.OneBillion < time.Now().Unix()
+	return rs+int64(preCommitStepTO)/constants.OneBillion < time.Now().Unix()
```

Comment:
Instead of converting with the constants.OneBillion use the constants out of the time package to perform conversion. This makes unit analysis easier for future readers. I will not block this PR for this, but please open a ticket in the backlog to fix in a future commit if no other changes are needed in subsequent files for PR to be merged.